docs/documentation/functions/certificates/Invoke-GeneratePrivateKeyAndCsr.md

```diff
@@ -0,0 +1,193 @@
+# Invoke-GeneratePrivateKeyAndCsr
+
+## Synopsis
+
+Generate the private key and certificate signing request (CSR) files using OpenSSL with a common name and additional parameters as necessary.
+
+## Syntax
+
+```powershell
+Invoke-GeneratePrivateKeyAndCsr [-outDirPath] <String> [-subjectAlternativeNames] <String> [-keySize] <Int>
+ [-expireDays] <Int> [-organization] <String> [-organizationUnit] <String> [-locality] <String>
+ [-state] <String> [-country] <String>
+```
+
+## Description
+
+The `Invoke-GeneratePrivateKeyAndCsr` cmdlet generates the private key and certificate signing request (CSR) files using OpenSSL with a common name and additional parameters as necessary.
+
+## Examples
+
+### Example 1
+
+```powershell
+Invoke-GeneratePrivateKeyAndCsr -outDirPath "C:\certificates" -commonName "sfo-vrli01.rainpole.io" -subjectAlternativeNames "sfo-vrli01a.rainpole.io, sfo-vrli01b.rainpole.io, sfo-vrli01c.rainpole.io" -keySize 4096 -expireDays 720 -organization "rainpole" -organizationUnit "IT" -locality "San Francisco" -state "California" -country "US"
+```
+
+This example will generate a private key and CSR pair for VMware Aria Operations for Logs where the integrated load balancer has a common name of sfo-vrli01.rainpole.io and includes SANs for each cluster node's fully qualified domain name value. 
+
+## Parameters
+
+### -outDirPath
+
+The directory path to store the signed certificate, private key and certificate signing request file.
```

Comment:
```suggestion
The directory path to store the signed certificate, private key, and certificate signing request file.
```

Reply:
addressed in commit c96a0aa15eab993f1b7f284e5e402712b1a9401f